Validator/IsJson.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Web.Script.Serialization;
+
+namespace Validator
+{
+    public partial class Validator
+    {
+        public static bool IsJson(string input)
+        {
+            if (String.IsNullOrEmpty(input))
+            {
+                return false;
+            }
+            try
+            {
+                JavaScriptSerializer serializer = new JavaScriptSerializer();
+                serializer.Deserialize<Dictionary<string, object>>(input);
```

Comment:
I think you can simplify this line to be `serializer.Deserialize<dynamic>(input)` or `serializer.Deserialize<object>(input)` (whichever really) and the tests will still pass.